ui/__tests__/util/policy.test.js

```diff
@@ -0,0 +1,48 @@
+import Policy from '../../util/policy';
+
+function policy(props = {}) {
+  return new Policy({
+    id: 1,
+    has_docnode: false,
+    omb_policy_id: 'M-16-19',
+    ...props,
+  });
+}
+
+describe('Policy', () => {
+  it('is frozen', () => {
+    expect(Object.isFrozen(policy())).toBe(true);
+  });
+
+  it('copies properties from passed-in object', () => {
+    expect(policy({ id: 4 }).id).toBe(4);
+  });
+
+  it('Serializes (via JSON) to the original passed-in object', () => {
```

Comment:
Honestly there's no strong reason that I added this; I just wanted the `Policy` class to be as close to the struct in behavior as possible, I guess.